tests/integration_tests/csv_storage_column_selector/run.sh

```
@@ -0,0 +1,59 @@
+		--upstream-uri "root@tcp(${UP_TIDB_HOST}:${UP_TIDB_PORT})/" \
+		--downstream-uri "root@tcp(${DOWN_TIDB_HOST}:${DOWN_TIDB_PORT})/" \
+		--databases "test,test1" \
+		--config="$CUR/conf/changefeed.toml"
+}
+
+function run() {
+	if [ "$SINK_TYPE" != "storage" ]; then
+		return
+	fi
+
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)